extensions/icons8/README.md

```diff
@@ -0,0 +1,28 @@
+<div align="center">
+  <img src="./media/Icons8-Art.svg" style="width: 96px; border-radius: 32px;"/>
```

Comment:
we current have a bug that doesn't allow us to render images in the README with relative paths. you will need to use the github absolute path here in order for this to work. also you could delete the media folder and have everything in assets.

Reply:
Hey I couldn't get it to work. How do you get the GitHub absolute path?